mobile/lib/modules/home/ui/control_bottom_app_bar.dart

```diff
@@ -51,73 +49,73 @@ class ControlBottomAppBar extends ConsumerWidget {
+            iconData: Icons.share_rounded,
             label: "control_bottom_app_bar_share".tr(),
-            onPressed: enabled ? onShare : null,
+            onPressed: enabled ? () => onShare(false) : null,
           ),
-          if (hasRemote)
-            ControlBoxButton(
-              iconData: Icons.archive,
-              label: "control_bottom_app_bar_archive".tr(),
-              onPressed: enabled ? onArchive : null,
-            ),
-          if (hasRemote)
-            ControlBoxButton(
-              iconData: Icons.favorite_border_rounded,
-              label: "control_bottom_app_bar_favorite".tr(),
-              onPressed: enabled ? onFavorite : null,
-            ),
+        ControlBoxButton(
+          iconData: Icons.ios_share_rounded,
+          label: "control_bottom_app_bar_share_to".tr(),
+          onPressed: enabled ? () => onShare(true) : null,
+        ),
+        if (hasRemote)
           ControlBoxButton(
-            iconData: Icons.delete_outline_rounded,
-            label: "control_bottom_app_bar_delete".tr(),
-            onPressed: enabled
-                ? () {
-                    if (!trashEnabled) {
-                      showDialog(
-                        context: context,
-                        builder: (BuildContext context) {
-                          return DeleteDialog(
-                            onDelete: onDelete,
-                          );
-                        },
-                      );
-                    } else {
-                      onDelete();
-                    }
+            iconData: Icons.archive,
+            label: "control_bottom_app_bar_archive".tr(),
+            onPressed: enabled ? onArchive : null,
+          ),
+        if (hasRemote)
+          ControlBoxButton(
+            iconData: Icons.favorite_border_rounded,
+            label: "control_bottom_app_bar_favorite".tr(),
+            onPressed: enabled ? onFavorite : null,
+          ),
+        ControlBoxButton(
+          iconData: Icons.delete_outline_rounded,
+          label: "control_bottom_app_bar_delete".tr(),
+          onPressed: enabled
+              ? () {
+                  if (!trashEnabled) {
+                    showDialog(
+                      context: context,
+                      builder: (BuildContext context) {
+                        return DeleteDialog(
+                          onDelete: onDelete,
+                        );
+                      },
+                    );
+                  } else {
+                    onDelete();
                   }
+                }
+              : null,
+        ),
+        if (!hasLocal)
+          ControlBoxButton(
+            iconData: Icons.filter_none_rounded,
+            label: "control_bottom_app_bar_stack".tr(),
+            onPressed: enabled ? onStack : null,
+          ),
+        if (!hasRemote)
+          ControlBoxButton(
+            iconData: Icons.backup_outlined,
+            label: "Upload",

```

Comment:
@shenlong-tanwen Could you please add translation of this button as well?

Reply:
Sure, I'll add it in any one of my upcoming PRs :D